WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -463,17 +470,23 @@ class MySiteViewModel @Inject constructor(
         }.toList()
     }
 
-    private fun scrollToQuickStartTaskIfNecessary(
-        quickStartTask: QuickStartTask?,
+    private fun scrollToQuickStartTask(
```

Comment:
@AjeshRPai - ❓ Why did you remove the `IfNecessary` part from the fun name? It's a pattern generally used throughout the app to denote whether the action is required.